### ISSUE TITLE
can't configure

### ISSUE THREAD

**Issue by Avpman2** [bentekkie/ha-generac#87]
Installed as per instructions. Most recent version online.
When I try to configure it, I get this error:
![image](https://github.com/user-attachments/assets/61797bd1-4636-45af-b701-b405f25bfbbd)
I can't get any logs. Sorry, please be patient with me I am still very new to HA.

## Configuration

```yaml
Add your logs here.
```

## Describe the bug

When trying to configure with my username and password (verified they are correct) I get the error in the screenshot.

## Debug log

```text

Add your logs here.

```

**Comment by Avpman2** [comment]
Logs attached.
[home-assistant_2024-09-22T23-57-03.854Z.log](https://github.com/user-attachments/files/17091683/home-assistant_2024-09-22T23-57-03.854Z.log)

**Comment by binarydev** [comment]
@Avpman2 if you haven't resolved this yet, you can try removing this custom integration and using the one that's part of the official HACS index of integrations, which is backed by my fork of this repo over at https://github.com/binarydev/ha-generac This is a dupe issue of #56

**Comment by Avpman2** [comment]
Thank you. I got it working. I'm only using the integration to monitor my propane take usage. I don't have a generator (yet). Is there a benefit to switching?

**Comment by Avpman2** [comment]
Thank you. I got it working. I'm only using the integration to monitor my propane take usage. I don't have a generator (yet). Is there a benefit to switching?

**Comment by Avpman2** [comment]
p.s. I don't see a Generac integration in HACS. Is it under a different name?

**Comment by binarydev** [comment]
Nope it's listed in the Community Store as "generac" and can be accessed directly via this link: https://my.home-assistant.io/redirect/hacs_repository/?owner=binarydev&repository=ha-generac&category=Integration

The only benefits of switching at the moment are:

1. propane tank monitors being supported in my fork, which aren't supported in this original report. You can compare the list of sensors at the top of my README to those in this repo's README to see if they matter to you or not.

2. wider community adoption and possible contribution levels since it's officially listed in the store and appears without needing to add a custom repo

**Comment by Avpman2** [comment]
looks like I'm already using your implementation but I installed it manually. I must have landed on your fork (ouch) when I couldn't get the original version to work. A couple of updates came out recently. Maybe one yesterday?

**Comment by Avpman2** [comment]
looks like I'm already using your implementation but I installed it manually. I must have landed on your fork (ouch) when I couldn't get the original version to work. A couple of updates came out recently. Maybe one yesterday?

**Comment by binarydev** [comment]
Yup a release was pushed in the last day or two to fix broken entities due to the 2025.1 release (plus a beta release to test a fix for an open issue)

**Comment by Avpman2** [comment]
Then I'm good! Thank you for this implementation and for reaching out to me.

**Comment by binarydev** [comment]
No prob, happy new year!

**Comment by Avpman2** [comment]
Thank you! To you and your family as well.
